.github/workflows/cura-installer.yml

```
@@ -296,9 +308,9 @@ jobs:
                 run: python ../cura_inst/packaging/AppImage/create_appimage.py ./UltiMaker-Cura $CURA_VERSION_FULL "${{ steps.filename.outputs.FULL_INSTALLER_FILENAME }}"
                 working-directory: dist
 
-            -   name: Create the MacOS dmg (Bash)
-                if: ${{ inputs.installer && runner.os == 'Macos' }}
-                run: python ../cura_inst/packaging/dmg/dmg_sign_noterize.py ../cura_inst . "${{ steps.filename.outputs.FULL_INSTALLER_FILENAME }}"
+            -   name: Create the MacOS pkg (Bash)
+                if: ${{ github.event.inputs.installer == 'true' && runner.os == 'Macos' }}
+                run: python ../cura_inst/packaging/MacOs/build_macos.py . "${{ steps.filename.outputs.FULL_INSTALLER_FILENAME }}"
```

```suggestion
                run: python ../cura_inst/packaging/MacOS/build_macos.py . "${{ steps.filename.outputs.FULL_INSTALLER_FILENAME }}"
```

And ofcourse change the name of the actual folder as well.